functions/backup.bash

```diff
@@ -69,9 +67,13 @@ restore_openhab_config() {
   fi
 
   echo -n "$(timestamp) [openHABian] Restoring openHAB backup... "
-  if ! cond_redirect systemctl stop openhab.service; then echo "FAILED (stop openHAB)"; return 1; fi
-  if ! (yes | cond_redirect openhab-cli restore "$filePath"); then echo "FAILED (restore)"; return 1; fi
-  if cond_redirect systemctl restart openhab.service; then echo "OK"; else echo "FAILED (restart openHAB)"; return 1; fi
+  if [[ "$1" == "--textonly" ]]; then
+    if ! (yes | cond_redirect "${OPENHAB_RUNTIME:-/usr/share/openhab/runtime}/bin/restore --textonly $filePath"); then echo "FAILED (restore)"; return 1; fi
```

Comment:
Can this not be called from the openhab-cli? If it can I would much prefer to use that then a hard link.

Also the quoting on this line is a little hard to read might be better to go with:
```suggestion
    if ! (yes | cond_redirect "${OPENHAB_RUNTIME:-/usr/share/openhab/runtime}"/bin/restore --textonly "$filePath"); then echo "FAILED (restore)"; return 1; fi
```

Reply:
openhab-cli would enforce openHAB being taken down but that's a major point here: importing text only allows for keeping OH running.

Reply:
Ahh, that makes sense. thanks for clarifying.